api/src/test/java/org/pmiops/workbench/auth/DelegatedUserCredentialsTest.java

```diff
@@ -38,8 +42,9 @@
   static final String USER_EMAIL = "john.doe@researchallofus.org";
   static final String SERVICE_ACCOUNT_EMAIL = "gsuite-admin@test-project.iam.gserviceaccount.com";
   static final List<String> SCOPES = Arrays.asList("openid", "profile");
+  static final String MOCK_ACCESS_TOKEN = "access-token";
```

Comment:
Is this what access tokens look like? It's fine for test purposes, but I find it easier to pick up a new class if the test methods use data that look like the real thing.